crates/standalone/src/main.rs

```diff
@@ -46,6 +46,15 @@ use tikv_jemallocator::Jemalloc;
 #[global_allocator]
 static GLOBAL: Jemalloc = Jemalloc;
 
+// Defaults our jemalloc configuration to allow for profiling, but have it disabled initially.
+// It can be enabled with an internal endpoint.
+// When enabled, it will sample once per 2^19 bytes (512 KB) of memory allocated.
+
+// This can be overridden by setting the `_RJEM_MALLOC_CONF` environment variable.
+#[allow(non_upper_case_globals)]
+#[export_name = "_rjem_malloc_conf"]
+pub static _rjem_malloc_conf: &[u8] = b"prof:true,prof_active:false,lg_prof_sample:19\0";
```

Comment:
is this basically a special specific name that jemalloc will look for? or where does this get used?

Reply:
Yes, jemalloc looks this up somehow.